kroto-plus-coroutines/src/test/kotlin/com/github/marcoferrer/krotoplus/coroutines/server/ServerCallBidiStreamingTests.kt

```diff
@@ -93,9 +96,11 @@ class ServerCallBidiStreamingTests {
             ) {
                 reqChannel = requestChannel
                 respChannel = responseChannel
-                requestChannel.mapTo(responseChannel) {
-                    HelloReply.newBuilder().setMessage("Reply: ${it.name}").build()
-                }
+                requestChannel
+                    .consumeAsFlow()
+                    .collect {
+                        responseChannel.send(HelloReply.newBuilder().setMessage("Reply: ${it.name}").build())
+                    }
```

Comment:
The `mapTo` operator on channel is now deprecated. With out this refactor, the test would fail.